docs/ConfigurationGuide.md

```diff
@@ -125,12 +125,33 @@ and at runtime when the user navigates to the QR code page, the proof would incl
 See the `oidc-controller\api\verificationConfigs\variableSubstitutions.py` file for implementations.
 
 #### Customizing variables
+For user defined variable substitutions users can set the environment
+variable to point at a python file defining new substitutions.
 
-In `oidc-controller\api\verificationConfigs\variableSubstitutions.py` there are the built-in variables above.
-For an advanced use case, if you require further customization, it could be possible to just replace that `variableSubstitutions.py` file 
-in a VC-AuthN implementation and the newly introduced variables would be run if they are included in a proof request configuration.
+##### User Defined Variable API
+In `oidc-controller\api\verificationConfigs\variableSubstitutions.py`
+you will find the method `add_variable_substitution` which can be used
+to modify the existing instance of `VariableSubstitutionMap` named
+`variable_substitution_map`.
```

Comment:
Wondering if we need to explicitly call-out how a substitution is loaded, if all a user need to do is provide a well-structured file with new substitutions?